addons/l10n_latam_check/wizards/account_payment_register.py

```diff
@@ -42,3 +50,8 @@ def _create_payment_vals_from_wizard(self, batch_result):
                 'l10n_latam_move_check_ids': [Command.link(x.id) for x in self.l10n_latam_move_check_ids]
             })
         return vals
+
+    def action_create_payments(self):
+        if self._is_latam_check_payment(check_subtype='move_check') and len(self.l10n_latam_move_check_ids.mapped('currency_id')) > 1:
+            raise ValidationError(_("You can't mix checks with different currencies. Please create separate payments for checks of each currency."))
```

Comment:
Do you think we should also check if `self.l10n_latam_move_check_ids.mapped('currency_id')` is equal to `self.currency_id` in this method? Currently we rely on the compute doing that, but it shouldn't be possible to select an ARS check and then change the currency to in the wizard to USD right?